### PR TITLE
configcat-feature-flag-sync 0.0.1

### PR DESCRIPTION
**Pull request by kp-cat**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/configcat-feature-flag-sync/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    is_required: true
+    summary: Full domain name without protocol e.g. 'api.configcat.com'
+    title: ConfigCat Public Management API host
+- configcat_api_user: $CONFIGCAT_API_USER
+  opts:
+    is_required: true
+    is_sensitive: true
+    summary: Create your ConfigCat Public Management API credentials [here](https://app.configcat.com/my-account/public-api-credentials/)
+    title: ConfigCat Public Management API basic authentication username
+- configcat_api_pass: $CONFIGCAT_API_PASS
```
Comment (godrei) [inline steps/configcat-feature-flag-sync/0.0.1/step.yml:51]: Same as for `configcat_api_user` input.

Comment (godrei) [inline steps/configcat-feature-flag-sync/0.0.1/step.yml:51]: Same as for `configcat_api_user` input.

File: steps/configcat-feature-flag-sync/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    title: 'Code snippet line count before and after the reference line (min: 1, max:
+      10)'
+- opts:
+    summary: e.g. 'subfolder/to/scan'
+    title: Sub-folder to scan, relative to the repository root folder
+  sub_folder: null
+- opts:
+    summary: '`true` or `false`'
+    title: Turns on detailed logging
+  verbose: false
```
Comment (godrei) [inline steps/configcat-feature-flag-sync/0.0.1/step.yml:76]: Step Input Value Options could be used here to ease this input configuration:

```
      value_options:
        - "false"
        - "true"
```

Please surround boolean values with double quotes (both for the value options and default value), to enforce string types.

Comment (godrei) [inline steps/configcat-feature-flag-sync/0.0.1/step.yml:76]: Step Input Value Options could be used here to ease this input configuration:

```
      value_options:
        - "false"
        - "true"
```

Please surround boolean values with double quotes (both for the value options and default value), to enforce string types.

Comment (godrei) [reply steps/configcat-feature-flag-sync/0.0.1/step.yml:76]: By using Value Options, no need for explaining the available values in the `summary`.
I propose something like:

```
- opts:
     summary: 'Turns on or off detailed logging'
     title: Detailed logging
      value_options:
        - "false"
        - "true"
   verbose: "false"
```

Comment (godrei) [reply steps/configcat-feature-flag-sync/0.0.1/step.yml:76]: By using Value Options, no need for explaining the available values in the `summary`.
I propose something like:

```
- opts:
     summary: 'Turns on or off detailed logging'
     title: Detailed logging
      value_options:
        - "false"
        - "true"
   verbose: "false"
```

**Review by godrei — CHANGES_REQUESTED** [review]
Hi @kp-cat ,

thanks for the new Step, I especially like the detailed Step description!

I left a few comments, please have a look at them.

Please follow [this guide](https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/sharing-steps-with-all-bitrise-users.html#fixing-issues-in-a-steplib-pull-request), while updating the step.

File: steps/configcat-feature-flag-sync/0.0.1/step.yml (hunk 2 above)
Comment (kp-cat) [reply steps/configcat-feature-flag-sync/0.0.1/step.yml:76]: Thanks for the comments. I fixed the issues and opened a new PR with version 0.0.2.

**Pull request closed without merging**